pages/common/kill.md

```diff
@@ -2,6 +2,7 @@
 
 > Sends a signal to a process, usually related to stopping the process.
 > All signals except for SIGKILL and SIGSTOP can be intercepted by the process to perform a clean exit.
+> More information: <https://manned.org/kill>.
```

Comment:
Did you see the GNU link?
```suggestion
> More information: <https://www.gnu.org/software/coreutils/manual/html_node/kill-invocation.html>.
```

Reply:
Yes. But I think the manned.org one is more organized and detailed.

Reply:
Looks like the version of `kill` installed on most Linux distros isn't from GNU, but from util-linux. I don't know about OSX tough.